tests/integrations/deploy/test_sf_backend.py

```diff
@@ -47,11 +51,10 @@ def test_integration_deploy_existing_function_serverless_backend(
     # Deploy twice in a row
     url, *_ = run_deploy_command(
         client,
-        app_path=constants.APP_FIXTURES_PATH.joinpath("app_updated.py"),
+        app_path=constants.APP_FIXTURES_PATH / "app_updated.py",
     )
 
-    import time
-
+    # TODO?: delete this. It's a kubernetes side-effect.
     time.sleep(30)
```

Comment:
This test is flaky, unfortunately :( Sometimes the updated function is marked as ready but it's older version will still respond to requests for up to a ~~minute.

Reply:
It's very random though which is why the CI passed when we merged those tests in #55.

Reply:
Can we do the same trick of retrying every 5 seconds for 50 seconds and only passing if it's the updated function response?